### ISSUE TITLE
Error about src/meshfem3D/model_aniso_mantle.f90

### ISSUE THREAD

**Issue by yanjia7707** [geodynamics/specfem3d_globe#744]
Hi,

I would like to set up my own mantle azimuthal anisotropic model. The manual said that I need to edit the program named "model_aniso_mantle.f90". As the first step, I changed the model type to "3D_anisotropic" in the Par_file without modifying the model_aniso_mantle.f90. However, an error named "Caught signal 8 (Floating point exception: floating-point divide by zero)" stoped the program. May I ask what is the error means and why it appears? I didn't do any modification in "model_aniso_mantle.f90" yet. May I ask if there are any mistakes in the original files? Thanks a lot!

> MODEL                           = 3D_anisotropic

```
starting MPI mesher on 1 processors
[compute-11-3:8544 :0:8544] Caught signal 8 (Floating point exception: floating-point divide by zero)
==== backtrace (tid:   8544) ====
 0  /usr/lib/libucs.so.0(ucs_handle_error+0x124) [0x7f897c90ee04]
 1  /usr/lib/libucs.so.0(+0x2422c) [0x7f897c90f22c]
 2  /usr/lib/libucs.so.0(+0x245ba) [0x7f897c90f5ba]
 3  */specfem3d_globe/EXAMPLES/111_3d-aniso-test2/bin/xmeshfem3D() [0x44d4dd]
 4  */specfem3d_globe/EXAMPLES/111_3d-aniso-test2/bin/xmeshfem3D() [0x473e27]
 5  */specfem3d_globe/EXAMPLES/111_3d-aniso-test2/bin/xmeshfem3D() [0x462cd9]
 6  */specfem3d_globe/EXAMPLES/111_3d-aniso-test2/bin/xmeshfem3D() [0x4037fe]
 7  /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xe7) [0x7f8980c30bf7]
 8  */specfem3d_globe/EXAMPLES/111_3d-aniso-test2/bin/xmeshfem3D() [0x40384a]
=================================
Program received signal SIGFPE: Floating-point exception - erroneous arithmetic operation.

Backtrace for this error:
#0  0x7f898101297f in ???
        at /build/glibc-S9d2JN/glibc-2.27/nptl/../sysdeps/unix/sysv/linux/x86_64/sigaction.c:0
#1  0x44d4dd in check_mesh_resolution_
        at src/meshfem3D/check_mesh_resolution.f90:170
#2  0x473e26 in create_regions_mesh_
        at src/meshfem3D/create_regions_mesh.F90:250
#3  0x462cd8 in create_meshes_
        at src/meshfem3D/create_meshes.f90:161
#4  0x4037fd in xmeshfem3d
        at src/meshfem3D/meshfem3D.f90:381
#5  0x4037fd in main
        at src/meshfem3D/meshfem3D.f90:389
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun noticed that process rank 0 with PID 0 on node compute-11-3 exited on signal 8 (Floating point exception).
--------------------------------------------------------------------------
```

**Comment by danielpeter** [comment]
thanks for pointing out! the 3D_anisotropic model wasn't supposed to run without a 3D crustal model. I'll fix that such that your setup can run.

**Comment by danielpeter** [comment]
please check with the latest devel version - should have been fixed by PR #745

**Comment by yanjia7707** [comment]
OK! I will check the modification, thanks for the help!
